README.md

```diff
@@ -17,7 +17,113 @@ TerseTS can be compiled and cross-compiled from source:
    - Microsoft Windows: `zig build -Dtarget=x86_64-windows -Doptimize=ReleaseFast`
 
 # Usage
-TODO
+
```

Comment:
Maybe start with how to use it from Zig to match the **Bindings** section.

Reply:
Completly changed following previous comments.